src/buildstream_plugins/elements/autotools.yaml

```diff
@@ -93,6 +94,8 @@ config:
   - |
     %{autogen}
   - |
+    mkdir -p "%{build-dir}"
+    cd "%{build-dir}"
     %{configure}
```

Comment:
Don't we need a `cd -` here?

Reply:
Turns out we don't. These aren't actually concatenated into one shell script.

Reply:
@abderrahim did I misunderstand what you were after?

Reply:
Indeed we do not need `cd -`.